hack/update-codegen.sh

```diff
@@ -17,33 +17,35 @@
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 PACKAGE_BASE=${PACKAGE_BASE:-"github.com/kube-node/nodeset"}
 CLIENT_PATH=pkg/client
-CLIENT_NAME=clientset_v1alpha1
+CLIENT_NAME=clientset
```

Comment:
No, still versioned. But if we get other versions as well, they will be part of the same clientset. At least that was the plan.